packages/discord.js/src/util/RoleFlagsBitField.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const { RoleFlags } = require('discord-api-types/v10');
+const BitField = require('./BitField');
+
+/**
+ * Data structure that makes it easy to interact with a {@link Role#flags} bitfield.
+ * @extends {BitField}
+ */
+class RoleFlagsBitField extends BitField {
+  /**
+   * Numeric role flags.
+   * @type {RoleFlags}
+   * @memberof RoleFlagsBitField
+   */
+  static Flags = RoleFlags;
```

Comment:
Well kts v13
so maybe look at other bit fields

Reply:
This PR is for v14 tho 🤔